src/block/mod.rs

```diff
@@ -116,11 +118,11 @@ impl std::fmt::Debug for Block {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         write!(
             f,
-            "Block {{ height: {}, len: {}, state: {}, parent: {} }}",
+            "Block {{ height: {}, len: {}, state: {}, parent: {}}}",
```

Comment:
Please keep this space and add the slot number. We want the slot number in the `Block` and `BlockWithoutHeight` structs

Reply:
@Isaac-DeFrain when adding `pub global_slot_since_genesis: GlobalSlotNumberV1,` to the `Block` struct, I get that the Hash trait is not implemented for the GlobalSlotNumberV1 type. Similar issue throughout, such as impl Debug for BlockWithoutHeight struct. This is relatively related to what I wrote above about whether or not we want to implement the Copy trait U32Json, which originates from the mina-rs dependecy / submodule.